crates/nsed-cli/examples/langchain_agent_sim.py

```
@@ -0,0 +1,80 @@
+    result = {"evaluations": evaluations}
+else:
+    print(f"Error: unknown phase '{phase}'", file=sys.stderr)
+    sys.exit(1)
+
+print("___NSED_START___")
+print(json.dumps(result))
+print("___NSED_END___")
+
+# ── Post-response framework noise ────────────────────────────────────────────
+print("Chain completed successfully.")
+print(f"Total tokens used: 1234")
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)